unison-src/transcripts/alias-many.output.md

```diff
@@ -46,33 +46,23 @@ Let's try it!
   
   Tip: You can use `undo` or `reflog` to undo this change.
 
-.> cd .mylib
-
-.mylib> find
+.> find-in mylib
 
   1.  stuff.List.adjacentPairs : [a] -> [(a, a)]
```

Comment:
My only question is whether this should be as-is, or `mylib.stuff.List.adjacentPairs`.

E.g. I can see people running `find-in foo.bar blah` then `move.term 1 new.namespace.term`, which won't work if we don't qualify the find results.

Alternatively we could display it unqualified, but have it fully qualified within the numbered args.

Reply:
I'm realizing this is the same situation we have now — where `cd` after `find` lead to potentially broken numbered args.

Reply:
I updated to fix this, although the prefixing implementation is a little funky